packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -195,6 +195,22 @@ class DirectStripeRoutes {
       );
     }
 
+    // Get the country from the payment details.
+    // There's only one charge (the latest), per Stripe's docs.
+    let sourceCountry;
+    const paymentMethodDetails =
+      subscription.latest_invoice.payment_intent.charges.data[0]
+        .payment_method_details;
+
+    if (
+      paymentMethodDetails &&
```

Comment:
Hmm, someday this might be a nice area to clean up when we're further into TypeScript and can use optional chaining